docs/book/src/topics/capibmadm/index.md

```diff
@@ -4,6 +4,12 @@ Kubernetes Cluster API Provider IBM Cloud Management Utility
 
 ## Install capibmadm
 
+<div class="tabs">
```

Comment:
you should follow something like this - https://github.com/kubernetes-sigs/cluster-api/blob/5857f3f4dfbb4a6ae26a877a1ff4c0c402e69a7e/docs/book/src/user/quick-start.md?plain=1#L75